interface/billing/search_payments.php

```diff
@@ -468,13 +468,13 @@ function SearchPayingEntityAction()
           </tr>
           <tr>
             <td align="right"></td>
-			<td align="left" class="text"><?php echo htmlspecialchars( xl('Payment From'), ENT_QUOTES).':' ?></td>
+			<td align="left" type="text" class="text"><?php echo htmlspecialchars( xl('Payment From'), ENT_QUOTES).':' ?></td>
```

Comment:
I agree. Need to re-style the 3 pages and get rid of using tables to format display which unfortunately is the default method of formatting pages in openEMR.  This represents an exciting opportunity to make more pages responsive and a lot of work 😀